src/game/world.rs

```diff
@@ -84,6 +104,48 @@ impl World {
         // );
     }
 
+    pub fn update(&mut self, input: &InputHandler, player_position: WorldCoordinate) {
+        if cfg!(feature = "chunk_loading") {
+            // (un?)load chunks as the player moves
+            let player_chunk = ChunkGridCoordinate::from_world_coordinate(player_position);
+            let range = self.render_distance;
```

Comment:
fixing

```
cannot borrow `self.chunks` as mutable because it is also borrowed as immutable

mutable borrow occurs hererustc(E0502)
world.rs(112, 24): immutable borrow occurs here
world.rs(113, 27): first borrow occurs due to use of `self` in closure
world.rs(115, 13): mutable borrow occurs here
world.rs(116, 18): immutable borrow later used by call
```